config/bootstrap.php

```diff
@@ -107,6 +107,7 @@
 if (Configure::read('debug')) {
     Configure::write('Cache._cake_model_.duration', '+2 minutes');
     Configure::write('Cache._cake_core_.duration', '+2 minutes');
+    Configure::write('Cache._cake_routes_.duration', '+2 minutes');
```

Comment:
Should this be lower? I could see a 2 min duration getting tedious when doing development.  What do you think of 30 seconds?

Reply:
Is there any value of caching it during development? Considering how annoying it can be sometimes

Reply:
Duration of 1-5 seconds? might be small enough to not get in the way.

Reply:
fixed, would it be a reason too to reduce debug mode cache times for _core_ and _model_ ?